boards/adafruit-metro-rp2040/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "adafruit-metro-rp2040"
+version = "0.1.0"
+authors = ["Thad House <thadhouse1@gmail.com>", "The rp-rs Developers"]
+edition = "2018"
+homepage = "https://github.com/rp-rs/rp-hal-boards/tree/main/boards/adafruit-metro-rp2040"
+description = "Board Support Package for the Adafruit Metro RP2040"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rp-rs/rp-hal-boards.git"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cortex-m = "0.7.2"
+rp2040-boot2 = { version = "0.2.0", optional = true }
+rp2040-hal = { version = "0.8.0" }
```

Comment:
It seems that `rp2040-hal` is now at `0.9.0`, which is not backwards compatible in how `bsp_pins!` is used.

Reply:
Ok. I'll update this and my other PR.

Reply:
Updated.